### PR TITLE
Yair branch

### PR DESCRIPTION
**Pull request by YairFe**

### CODE REVIEW

File: screens/DSitterProfile.js [hunk 1]
```diff
@@ -1,59 +1,97 @@
...
-                        flex: 0.5,
-                        justifyContent: "space-evenly",
-                        alignItems: "center"
-                    }}>
-                        <View>
-                            <Text style={StyleSheet.compose(styles.statsText, {color: "#00c608"})}> 5$/Hour </Text>
+                        <BoxShadow setting={shadowInfo}>
+                            <View style={styles.informationBox}>
+                                    <Text style={styles.contentStyle}>
+                                        1.2km
```
Comment (Elad-Feldman) [inline screens/DSitterProfile.js:56]: looks good!
 later we would see how to take data from DB

File: screens/SignUpDSitterScreen.js [hunk 2]
```diff
@@ -75,7 +75,7 @@ const SignUpDSitterScreen = ({navigation}) => {
                     }/>
                 </View>
                 <View style={{flexDirection: 'row', justifyContent: 'space-evenly', marginTop: 25, marginBottom: 25}}>
-                    <NewButton title="Next" onPress={()=> {handleLogin(); navigation.navigate("Find your Care-Giver")}}/>
+                    <NewButton title="Next" onPress={()=> {/* handleLogin(); */ navigation.navigate("Find your Care-Giver")}}/>
```
Comment (Elad-Feldman) [inline screens/SignUpDSitterScreen.js:78]: what is that comment?
maybe not in the same line?

**Review by Elad-Feldman — DISMISSED** [review]
looks good, I think we could merge

File: App.js [hunk 3]
```diff
@@ -29,23 +29,32 @@ const headerTitleStyle = {
...
-        setTimeout(() => {
-            setLoading(false)
-        }, 1000);  //5000 milliseconds
+        const subscriber = auth().onAuthStateChanged((user) =>{
+            setUser(user);
+            if (isLoading) {
+                console.log(user);
+                setTimeout(() => {
+                    setLoading(false)
+                }, 1000);  //5000 milliseconds
```
[the remark this answers is not in the document]
Comment (YairFe) [reply App.js:44]: i copied daniel previous code idk either how maybe he changed it and forgot to change the comment

File: screens/SignUpDSitterScreen.js (hunk 2 above)
Comment (YairFe) [reply screens/SignUpDSitterScreen.js:78]: this function set the auth state daniel create to true and therefore the signup wont show i changed it so the logging information will be from the firebase

**Review by Elad-Feldman — APPROVED** [review]
good !

**Pull request merged**